createInitialDirectory.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+mkdir ~/.InitialProject
+cp -rpa ./InitialProject/* ~/.InitialProject/
+cp -rpa ./InitialProject/.gitignore ~/.InitialProject/
+chmod 777 ~/.InitialProject
+sudo cp InitProject /usr/local/bin/
+sudo chmod 777 /usr/local/bin/InitProject
```

Comment:
Need to add .sh file extension to InitProject

Reply:
Done